src/diagonal.works/b6/api/shell.go

```diff
@@ -566,6 +568,79 @@ func reduceSymbolsSymbols(ss []*pb.NodeProto, s *pb.NodeProto) []*pb.NodeProto {
 	return reduced
 }
 
+func reduceCollectionItems(collection *pb.NodeProto) *pb.NodeProto {
```

Comment:
i think reduce is descriptive enough / but up to u

Reply:
but maybe u cant reuse that for other methods / idk how same name / different param works in go

Reply:
Yeah, you can't reuse names at all, and "reduce" is the common term in a parser for for turning the right hand side of a production rule into the left, so it's used all over the place - hence I need to suffix everything.